.github/workflows/pr.yml

```diff
@@ -24,6 +24,9 @@ jobs:
             - name: Lint
               run: pnpm lint
 
+            - name: Format
+              run: pnpm exec prettier --check --ignore-path ./dev/configs/.prettierignore .
```

Comment:
Formatting is run automatically on commit. Is there a benefit to having a check here as well?

Reply:
I had the commit hook on and it wasn't working. It's also common that developers completely disable commit hooks.

Reply:
Makes sense, thanks! In that case, can we just add a new script called `checkFormat` so people can run it locally (e.g. probably should use that in prChecks), then use that in CI?